decidim-conferences/app/commands/decidim/conferences/admin/create_conference.rb

```diff
@@ -85,6 +89,34 @@ def send_notification
             recipient_ids: conference.followers.pluck(:id)
           )
         end
+
+        def participatory_processes(conference)
```

Comment:
No need to pass `conference` around as a parameter, you have access to it globally already.